nautobot_floor_plan/api/nested_serializers.py

```diff
@@ -6,7 +6,7 @@
 from nautobot_floor_plan import models
 
 
-class NestedFloorPlanSerializer(WritableNestedSerializer):
+class FloorPlanNestedSerializer(WritableNestedSerializer):
```

Comment:
wrong; revert. But why is a cookiecutter PR making changes to the source code at all?

Reply:
Reverted